src/components/Datepicker/Datepicker.js

```diff
@@ -1,39 +1,59 @@
 import React from 'react';
-import { Button, ButtonIcon } from 'react-lds';
+import { Button, ButtonIcon, Input } from 'react-lds';
 import classnames from 'classnames';
 
 import moment from 'moment';
 import 'moment-range';
 
 import { prefixClasses } from '../../utils';
 
+const iso8601DateFormat = 'YYYY-MM-DD';
+const salesforceDateFormat = 'M/D/YYYY';
+
 export class Datepicker extends React.Component {
   static contextTypes = { cssPrefix: React.PropTypes.string };
 
   static propTypes = {
     /**
-    * Optional preselected date
+    * Whether the datepicker is open or closed
     */
-    selectedDate: React.PropTypes.string,
+    isOpen: React.PropTypes.bool,
     /**
-    * Callback function once day has been changed
+    * Whether input is required
     */
-    onSelectDate: React.PropTypes.func.isRequired,
+    isRequired: React.PropTypes.bool,
     /**
-    * How many years to show before and after the current year
+    * Callback function if input is clicked
     */
-    yearSpan: React.PropTypes.number,
+    onToggle: React.PropTypes.func.isRequired,
+    /**
+    * Callback function once a valid date has been selected or input
+    */
+    onChangeValidDate: React.PropTypes.func.isRequired,
+    /**
+    * Optional preselected date
+    */
+    preselectedDate: React.PropTypes.string,
     /**
     * Optional translations, default to english
     */
     translations: React.PropTypes.shape({
+      inputFieldError: React.PropTypes.string,
+      inputFieldLabel: React.PropTypes.string,
       today: React.PropTypes.string,
     }),
+    /**
```

Comment:
Your comment indentation is incorrect, should be:

```js
/**
 * mops
 */
```

I suggest Atoms docblockr package.